README.md

```diff
@@ -434,7 +433,7 @@ If you'd like to customize this list, modify the example code in ```config/initi
 
 ```ruby
 config.exception_level_filters.merge!({
-  'ActiveRecord::RecordNotFound' => 'ignore',
+  'ActionController::RoutingError' => 'ignore',
```

Comment:
No, please see the PR comment. This was the second doc bug I fixed. See how the doc text says "to ignore routing errors..." and then the example is for RecordNotFound instead.